pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -50,6 +51,8 @@ const (
 	controllerWebhookPort = 9443
 	// common prefix for all resource of an operand
 	controllerResourcePrefix = "aws-load-balancer-controller"
+	// controllerReEnqueueDuration is the delay to re-enqueue.
+	controllerReEnqueueDuration = time.Second * 30
```

Comment:
To be precise this value is used when the secret is missing, for some other retry condition it could be something else. So a better name would be `secretMissingReEnqueueDuration`.

Reply:
Fixed.